Building DAG of jobs...
Using shell: /usr/bin/bash
Provided cores: 1 (use --cores to define parallelism)
Rules claiming more threads will be scaled down.
Job counts:
	count	jobs
	1	all
	72	verified_stats
	73
Select jobs to execute...

[Fri Dec  4 12:23:35 2020]
rule verified_stats:
    input: data/verified_vars/ENSSSCG00000015048.verified
    output: data/verified_stats/ENSSSCG00000015048.stats
    jobid: 121
    wildcards: id=ENSSSCG00000015048

[Fri Dec  4 12:23:36 2020]
Finished job 121.
1 of 73 steps (1%) done
Select jobs to execute...

[Fri Dec  4 12:23:36 2020]
rule verified_stats:
    input: data/verified_vars/ENSSSCG00000013369.verified
    output: data/verified_stats/ENSSSCG00000013369.stats
    jobid: 129
    wildcards: id=ENSSSCG00000013369

[Fri Dec  4 12:23:36 2020]
Finished job 129.
2 of 73 steps (3%) done
Select jobs to execute...

[Fri Dec  4 12:23:36 2020]
rule verified_stats:
    input: data/verified_vars/ENSSSCG00000001428.verified
    output: data/verified_stats/ENSSSCG00000001428.stats
    jobid: 181
    wildcards: id=ENSSSCG00000001428

[Fri Dec  4 12:23:37 2020]
Finished job 181.
3 of 73 steps (4%) done
Select jobs to execute...

[Fri Dec  4 12:23:37 2020]
rule verified_stats:
    input: data/verified_vars/ENSSSCG00000026583.verified
    output: data/verified_stats/ENSSSCG00000026583.stats
    jobid: 189
    wildcards: id=ENSSSCG00000026583

[Fri Dec  4 12:23:37 2020]
Finished job 189.
4 of 73 steps (5%) done
Select jobs to execute...

[Fri Dec  4 12:23:37 2020]
rule verified_stats:
    input: data/verified_vars/ENSSSCG00000016940.verified
    output: data/verified_stats/ENSSSCG00000016940.stats
    jobid: 69
    wildcards: id=ENSSSCG00000016940

[Fri Dec  4 12:23:38 2020]
Error in rule verified_stats:
    jobid: 69
    output: data/verified_stats/ENSSSCG00000016940.stats

RuleException:
CalledProcessError in line 49 of /home/pelmo/sscrofa_variant_remapping/Snakefile:
Command 'set -euo pipefail;  /home/pelmo/anaconda3/envs/snakemake/bin/python3.9 /home/pelmo/sscrofa_variant_remapping/.snakemake/scripts/tmp6e52p2k1.verified_stats.py' returned non-zero exit status 1.
  File "/home/pelmo/anaconda3/envs/snakemake/lib/python3.9/site-packages/snakemake/executors/__init__.py", line 2273, in run_wrapper
  File "/home/pelmo/sscrofa_variant_remapping/Snakefile", line 49, in __rule_verified_stats
  File "/home/pelmo/anaconda3/envs/snakemake/lib/python3.9/site-packages/snakemake/executors/__init__.py", line 560, in _callback
  File "/home/pelmo/anaconda3/envs/snakemake/lib/python3.9/concurrent/futures/thread.py", line 52, in run
  File "/home/pelmo/anaconda3/envs/snakemake/lib/python3.9/site-packages/snakemake/executors/__init__.py", line 546, in cached_or_run
  File "/home/pelmo/anaconda3/envs/snakemake/lib/python3.9/site-packages/snakemake/executors/__init__.py", line 2304, in run_wrapper
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
Complete log: /home/pelmo/sscrofa_variant_remapping/.snakemake/log/2020-12-04T122335.715890.snakemake.log
